Fixed issue 2104

ghankaremore opened the pull request:
Fixed the issue 2104 and implemeted stdev and stdev successfully

ghankaremore commented:
Zipped file

Please try to open this url in new tab.
[https://drive.google.com/file/d/1ltFiup04GEcG0c8HVvixtV4UzuRE09LD/view?usp=drive_link](url)

Report

[Siglens_Report.docx](https://github.com/user-attachments/files/18694124/Siglens_Report.docx)


Implemented spl query with stdev
![1](https://github.com/user-attachments/assets/06680aeb-23c6-4ea1-92cd-e0281794d799)
Implemented spl query with stdevp
![image](https://github.com/user-attachments/assets/11ea13f4-6bb3-4e75-b75c-327608b4f585)
Added stdev to aggregate attributes successfully
![image](https://github.com/user-attachments/assets/70b46da2-1928-428c-98c2-256f067430e4)
Added stdep to aggregate attributes successfully
![image](https://github.com/user-attachments/assets/13ebfdf5-0162-4a2f-83df-d23c9a3adb55)

nkunal commented:
This PR is not correct,
In this issue the actual mathematical functions for standard deviations were supposed to be implemented

ghankaremore commented:
Updated zip file 
[https://drive.google.com/file/d/1jdwCw7sYWqn2sYm74baOjEvaaJ5cTeV2/view?usp=sharing](url)

Updated report 

[Siglens_Report.docx](https://github.com/user-attachments/files/18699625/Siglens_Report.docx)
1 Implemented function for calculating stdev and stdevp in ssauges.go in instrumentation package
2 Added fields for standard deviation and population standard deviation in the NumericStats struct in segstructs.go in structs package. 
3 Implemented the functions to calculate standard deviation and population standard deviation in evalaggs.go in aggregation package. 
4 Removed stdev and stdevp from unsupportedStatsFuncs
5located calculations array and added stdev and stdevp.
